services/todoApi.ts

```diff
@@ -0,0 +1,15 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+import { Task } from "@/modules/HomePage/types/Task";
+
+export const todoApi = createApi({
+  reducerPath: "todoApi",
+  baseQuery: fetchBaseQuery({ baseUrl: "http://localhost:3001" }),
```

Comment:
Best practice is to put the url in your `.env` file and import that var from there.